### PR TITLE
Update main-button css

### PR DESCRIPTION
**Pull request by sparsh-99**
#156 
### New Features:-
1. Added the animations in the .main-button class
2. Updated the buttons to look it more attractive.

### CODE REVIEW

File: src/components/button/Button.css [hunk 1]
```diff
@@ -18,7 +18,31 @@
 .main-button:hover {
   background-color: #ffffff;
   color: black;
-  transition: ease-in 0.3s;
+  transition: ease-in 0.5s;
```
Comment (kartik918) [inline src/components/button/Button.css:21]: I see you've increased the transition time, we don't need slower animations so let's revert this.

Comment (sparsh-99) [reply src/components/button/Button.css:21]: Ok, thanks. Working on the outward animation
Also, I will revert the transition time.
Thanks

**Comment by sparsh-99** [comment]
Sorry @kartik918 @saadpasta , by mistake, I commit a new file here which has no reference here and I am unable to revert it back? Can you please help or delete that delete. It will be very helpful. Thank You

**Comment by saadpasta** [comment]
@sparsh-99 `git revert 7d6b832` should work I guess.

**Comment by sparsh-99** [comment]
Thank you @saadpasta , but due to some internal errors, the previous log was deleted. Now `git log` is not showing this commit. That's why I can't revert it back

File: src/components/githubRepoCard/GithubRepoCard.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default function GithubRepoCard({ repo }) {
     <div>
       <div className="repo-card-div" key={repo.node.id} onClick={() => openRepoinNewTab(repo.node.url)}>
         <div className="repo-name-div">
-          <svg aria-hidden="true" className="octicon" height="16" role="img" viewBox="0 0 12 16" width="12" className="repo-svg">
+          <svg aria-hidden="true" className="octicon" height="20" role="img" viewBox="0 0 12 16" width="14" className="repo-svg">
```
Comment (kartik918) [inline src/components/githubRepoCard/GithubRepoCard.js:16]: Remove changes to this file.

File: src/containers/projects/Project.css [hunk 3]
```diff
@@ -1,5 +1,5 @@
 .project-title {
-  font-size: 56px;
+  font-size: 52x;
```
Comment (kartik918) [inline src/containers/projects/Project.css:2]: Discard this file from the PR too.

**Comment by sparsh-99** [comment]
Hey @kartik918 @saadpasta , What if I again opened the PR for the same issue as reverting the last commit got some issues??
Thank You.

**Comment by kartik918** [comment]
If you would like to then it's fine, just do everything in one commit and reference the issue in the PR desc @sparsh-99

**Comment by sparsh-99** [comment]
Sure @kartik918 . Thank You.

**Pull request closed without merging**